.pre-commit-config.yaml

```diff
@@ -8,7 +8,6 @@ repos:
       rev: v4.5.0
       hooks:
           - id: check-yaml
-          - id: check-toml
```

Comment:
This is not needed, since `ruff` already parses `pyproject.toml` and prints nice errors if there is a problem.